lib/screens/about/states/about_state.dart

```diff
@@ -0,0 +1,12 @@
+import 'dart:core';
+
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'about_state.freezed.dart';
+
+@freezed
+class AboutState with _$AboutState {
+  const factory AboutState.loading() = _AboutStateLoading;
+  const factory AboutState.success() = _AboutStateSuccess;
+  const factory AboutState.failure(String error) = _AboutStateFailure;
```

Comment:
For this screen, I'm doubting the need of state, I like how we are following the stadars. I'm just thinking that this screen is very static, no need of endpoint request, we might not need to make it more verbose if it is gonna be as simple as just an Static UI

Reply:
At the moment what I did was create the mvvm pattern for the about screen, but the window has not been designed yet, also being a very basic screen that will not make any api requests I do not see that it is necessary to manage states